stratum/hal/lib/tdi/tdi_sde_helpers.cc

```diff
@@ -424,11 +432,18 @@ ::util::Status GetAllEntries(
     std::unique_ptr<::tdi::TableData> table_data;
     RETURN_IF_TDI_ERROR(table->keyAllocate(&table_key));
     RETURN_IF_TDI_ERROR(table->dataAllocate(&table_data));
-    RETURN_IF_TDI_ERROR(table->entryGetFirst(
-        *tdi_session, tdi_dev_target,
+    tdi_status = table->entryGetFirst(
+        *tdi_session, *dev_tgt,
         flags, table_key.get(),
-        table_data.get()));
-
+        table_data.get());
+    if (tdi_status != TDI_SUCCESS) {
+        //SDE iterates through all the tables, and if no entry is present for that table,
+        //TDI_OBJECT_NOT_FOUND is returned in which case it's no operation for flow dump for that table.
+        if (tdi_status == TDI_OBJECT_NOT_FOUND) {
+            return ::util::OkStatus();
+        }
+       RETURN_IF_TDI_ERROR(tdi_status);
```

Comment:
Coding guidelines: The closing brace and the following statement should be vertically aligned. They should be indented two spaces from the originating `if`.